plugins/sonarqube/api-report.md

```diff
@@ -21,17 +21,34 @@ export const EntitySonarQubeCard: (props: {
   duplicationRatings?: DuplicationRating[] | undefined;
 }) => JSX.Element;
 
+// @public (undocumented)
+export const EntitySonarQubeContentPage: (
+  props: SonarQubeContentPageProps,
+) => JSX.Element;
+
 // @public (undocumented)
 export const isSonarQubeAvailable: (entity: Entity) => boolean;
 
+// @public (undocumented)
+export const SONARQUBE_PROJECT_KEY_ANNOTATION = 'sonarqube.org/project-key';
+
 // @public (undocumented)
 export const SonarQubeCard: (props: {
   variant?: InfoCardVariants;
   duplicationRatings?: DuplicationRating[];
 }) => JSX.Element;
 
 // @public (undocumented)
-const sonarQubePlugin: BackstagePlugin<{}, {}, {}>;
-export { sonarQubePlugin as plugin };
-export { sonarQubePlugin };
+export const SonarQubeContentPage: (
```

Comment:
OK one final thing. This one should not be exported. It's only the provided version of it (EntitySonarQubeContentPage) that people should consume, so the "inner" one can stay private.

Reply:
done